### PR TITLE
silabs-multiprotocol: Use dynamic ingress port

### PR DESCRIPTION
**Pull request by agners**
Fixes: #2872

### CODE REVIEW

File: silabs-multiprotocol/translations/en.yaml [hunk 1]
```diff
@@ -39,4 +39,5 @@ configuration:
       Use OpenThread Border Router firewall to block unnecessary traffic.
 network:
   9999/tcp: EmberZNet EZSP/ASH port
+  8080/tcp: OpenThread Web port
```
Comment (frenck) [inline silabs-multiprotocol/translations/en.yaml:42]: Seems to be unused?

File: silabs-multiprotocol/config.yaml [hunk 2]
```diff
@@ -36,6 +36,7 @@ ports:
   8081/tcp: null
 ports_description:
   9999/tcp: EmberZNet EZSP/ASH port
+  8080/tcp: OpenThread Web port
```
Comment (frenck) [inline silabs-multiprotocol/config.yaml:39]: Seems to be unused?

**Review by frenck — CHANGES_REQUESTED** [review]

File: silabs-multiprotocol/translations/en.yaml (hunk 1 above)
Comment (agners) [reply silabs-multiprotocol/translations/en.yaml:42]: Right, I don't need that when allocating automatically. Will remove those.

**Review by frenck — APPROVED** [review]

**Comment by agners** [comment]
Since the Web interface requires the REST API as well (which doesn't go through ingress) we decided to remove ingress completely.

Instead we plan to start the web interface when the relevant ports (Web UI/REST API) are exposed externally.

**Pull request closed without merging**